irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -95,9 +95,13 @@ namespace iroha {
       st.exchange(soci::use(perm_str));
 
       auto msg = [&] {
+        const auto &str =
+            shared_model::proto::permissions::toString(permissions);
+        const auto perm_debug_str =
+            std::accumulate(str.begin(), str.end(), std::string());
```

Comment:
That would be barely readable because it doesn't have spaces, e.g would look like "can_add_peercan_create_domaincan_create_user"